qa/suites/rados/monthrash/ceph.yaml

```diff
@@ -7,3 +7,6 @@ overrides:
 tasks:
 - install:
 - ceph:
+# make sure mgr is up by flushing all pg stats
+- ceph_manager.flush_all_pg_stats:
```

Comment:
this can go away with the ceph.py patches that follow.  the old code only checked HEALTH_OK which didn't make us wait for the mgr to become available.